js/app/src/components/StatusTracker/ToastContent.svelte

```diff
@@ -0,0 +1,275 @@
+<script lang="ts">
+	import { Alert, Information, Warning } from "@gradio/icons";
+	import { createEventDispatcher, onMount, afterUpdate } from "svelte";
+	import { fade } from "svelte/transition";
+	import type { ToastMessage } from "./types";
+
+	export let message: string = "";
+	export let type: ToastMessage["type"];
+	export let id: number;
+
+	const dispatch = createEventDispatcher();
+
+	function close_message() {
+		dispatch("close", id);
+	}
+
+	onMount(() => {
+		setTimeout(() => {
+			close_message();
+		}, 10000);
+	});
+</script>
+
+<div
+	class="toast-body {type}"
+	role="alert"
+	on:click|stopPropagation
+	on:keydown|stopPropagation
+	in:fade={{ duration: 200, delay: 100 }}
+	out:fade={{ duration: 200 }}
+>
+	<div class="toast-icon {type}">
+		{#if type === "warning"}
+			<Warning />
+		{:else if type === "info"}
+			<Information />
+		{:else if type === "error"}
+			<Alert />
+		{/if}
+	</div>
+
+	<div class="toast-details {type}">
+		<div class="toast-title {type}">{type}</div>
+		<div class="toast-text {type}">
+			{message}
+		</div>
+	</div>
+
+	<button
+		on:click={close_message}
+		class="toast-close {type}"
+		type="button"
+		aria-label="Close"
+	>
+		<span aria-hidden="true">&#215;</span>
+	</button>
+
+	<div class="timer {type}" />
+</div>
+
+<style>
+	.toast-body {
+		display: flex;
+		position: relative;
+		top: var(--size-8);
+		right: 0;
+		left: 0;
+		align-items: center;
+		margin: var(--size-6) var(--size-4);
+		margin: auto;
+		border-radius: var(--container-radius);
+		max-width: 1200px;
+		max-width: 610px;
```

Comment:
Not sure what is going on with the duplication, but do we need this? The width is handled in the parent (or should be).